internal/ackhandler/interfaces.go

```diff
@@ -39,6 +39,9 @@ type SentPacketHandler interface {
 
 	GetAlarmTimeout() time.Time
 	OnAlarm() error
+
+	// report some congestion statistics. For tracing only.
+	GetCurrentState() *State
```

Comment:
Maybe call this stats instead, to make it more obvious that this is informational only?

Reply:
Done.